dev.sparcles.SREngine.yml

```diff
@@ -0,0 +1,43 @@
+id: dev.sparcles.SREngine
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: srengine-run.sh
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+  - --socket=pulseaudio
+  - --share=network
+  - --filesystem=home:create
+
+build-options:
+  build-args:
+    - --filesystem=home:create
+
+modules:
+   - name: srengine
+     buildsystem: cmake-ninja
+     config-opts:
+       - -DCMAKE_BUILD_TYPE=Release
+       - -DSR_ENGINE_FLATPAK_BUILD=ON
+       - -DSR_ENGINE_EXTENDED_NAME=OFF
+
+     build-commands:
+       - install -Dm755 Engine/srengine /app/bin/srengine
+       - install -Dm755 ../srengine-run.sh /app/bin/srengine-run.sh
+       - mkdir -p ~/.local/share/SREngine
+       - mv ../Resources ~/.local/share/SREngine
```

Comment:
You don't have access during build or after installation. You should make your application put this to `$XDG_DATA_HOME/SREngine`